test/support/api.ts

```diff
@@ -54,3 +60,8 @@ export class Api {
     return { [strategy]: apiKey };
   }
 }
+
+interface CreateApiResponse {
+  api: Api;
+  mockGraphClientService: MockGraphClientService;
+}
```

Comment:
I think `Response` in the interface name here is misleading - it makes it sounds like we're talking about a HTTP response (especially with `Api` just before it)